### ISSUE TITLE
Option: preload whole track to RAM

### ISSUE THREAD

**Issue by Soukyuu** [Alexey-Yakovenko/deadbeef#1570]
This is something foobar2000 allows and I find useful with libraries hosted on a network share. Basically, you have an option (under playback tab) to enable preloading files to RAM shortly before the player jumps to the next track, with a text box to specify the maximum size of a file to preload in MB.

In my experience, on cabled network it allows to cache even lossless files without any pause inbetween tracks. Main scenario is to avoid audio dropouts in case the network goes down for a short period of time, or after resuming from sleep (the remote server usually has invalidated your file handle by then).

**Comment by kevinlekiller** [comment]
If this is not possible, then maybe allow some of the cache settings to be changed, from what I can see it's about 500KB currently.

**Comment by anewuser** [comment]
@kevinlekiller The maintainer of the project has already added the "enhancement" tag to this issue, so it means that he has accepted it as a valid request. Now you have just to wait for him to have time to implement it, or submit a patch by yourself.

**Comment by Logicwax** [comment]
yeah is there anyway to configure this?   I can't seem to find any way to get deadbeef to preload the next track, which results in studdering when it moves to the next song.
